Dockerfile

```diff
@@ -1,17 +1,8 @@
-FROM node:14.16.0-buster
-
+FROM node:16-alpine
```

Comment:
why 16?

Reply:
14 was using llhttp 2, so I made sure to build using the same supported version.

Moreover, 14 is going EOL in 3 months so my lazy me took control and included the change in this PR as well.